### PR TITLE
#161254615 codeclimate badges

### PR DESCRIPTION
**Pull request by Eyansky**
#### What does this PR do?
- setup codeclimate badges.

#### Description of Task to be completed?
- Setup codeclimate badges
- setup codeclimate reviews for pull requests

#### How should this be manually tested?
- Check for review on this pull request from code climate.

#### Any background context you want to provide?
- I removed the comments from serviceworker.js file because codeclimate claimed the file had too many lines of code.
- the test badge is inactive because there are no tests yets.

#### What are the relevant pivotal tracker stories?
- [#161254615](https://www.pivotaltracker.com/story/show/161254615)

#### Screenshots (if appropriate)
screenshot 2018-11-28 at 11 30 20

#### Checklist:

- [x] My code follows the style guidelines of this project
- [x] At least 2 people have reviewed my PR
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] My PR has one commit.

### CODE REVIEW

**Review by abadojack — CHANGES_REQUESTED** [review]
Don't remove all the comments. You could summarize them at least

**Comment by Eyansky** [comment]
@abadojack  all your requested changes have been made.

File: src/serviceWorker.js [hunk 1]
```diff
@@ -132,4 +120,4 @@ export function unregister() {
       registration.unregister();
     });
   }
-}
+}
```
Comment (evamaina) [inline src/serviceWorker.js]: add new line here

**Comment by kipropbrian** [comment]
Looks good, although next time make sure your PR does what is advertised and nothing more. Try avoid adding extra changes.

**Review by evamaina — APPROVED** [review]

**Review by wrotich — APPROVED** [review]
LGTM

**Review by abadojack — APPROVED** [review]
LGTM

**Pull request merged**